src/boot/z_std_dma.c

```diff
@@ -11,7 +11,7 @@ s32 sDmaMgrCurFileLine;
 
 u32 D_80009460 = 0;
 u32 gDmaMgrDmaBuffSize = 0x2000;
-u32 sDmaMgrDataExistError = 0;
+u32 sDmaMgrIsRomCompressed = false;
```

Comment:
Since this seems to update per file it should probably be `IsFileCompressed`

Reply:
Usually we say a rom is decompressed if no files are compressed, so a rom is "compressed" if at least one file is compressed (also in compressed roms most files are typically compressed but not all)

What matters anyway is that at the end of `DmaMgr_ProcessMsg` this boolean is used if no relevant entry is found in the dma table containing the target vrom offset,
either to error if the rom is compressed (`sDmaMgrIsRomCompressed` is true),
or to do dma anyway assuming vrom = rom

and vrom = rom needs the rom to be "fully" decompressed to have a chance of being true (as a piece of trivia, vrom = rom isn't required otherwise though)